operator/pkg/cli/migrate_v2_pod.go

```diff
@@ -0,0 +1,339 @@
+package cli
```

Comment:
why did this file end up moving out of migratev2?

Reply:
it didnt. i changed it from "job" to "pod"

Reply:
That just says "it was unintentional" to me, though?

Reply:
It used to be in operator/pkg/cli/migratev2/managerjob.go

Reply:
This is a new pod that got added. managerjob.go changed to managerpod.go.

This file is covered in the bullet `kots will now run a pod for the migration rather than run it in process as the eco namespace gets removed and there are permissions errors`